client/src/App.jsx

```diff
@@ -1,21 +1,21 @@
-import React from 'react'
-import './App.css'
-import { useEffect, useState } from 'react'
-import { getRecipeByIngredientsService } from './service/recipe'
+import React, { useEffect, useState } from 'react';
+import './App.css';
+
+import { getRecipeByIngredientsService } from './service/recipe';
 
 function App() {
-    const [recipes, setRecipes] = useState([])
-    useEffect(() => {
-        getRecipeByIngredientsService(['apple', 'flour'], 2).then(r => setRecipes(r))
-    }, [])
+  const [recipes, setRecipes] = useState([]);
+  useEffect(() => {
+    getRecipeByIngredientsService(['apple', 'flour'], 2).then((r) => setRecipes(r));
```

Comment:
I'm assuming the `['apple', 'flour']` is just a temporary placeholder for now?

Reply:
This is an existing class that I only re-formatted, and we would be deleting it eventually.